bootstrap/bootstrap.go

```diff
@@ -379,6 +363,11 @@ func (b *Bootstrap) tearDown() error {
 		return err
 	}
 
+	// Support deprecated BUILDKITE_DOCKER* env vars
+	if hasDockerEnvironment(b.shell) {
```

Comment:
Can we call it `hasDeprecatedDockerEnvironment` just to make it clear that it shouldn't be considered the future?

Reply:
Hah yup, I called it that initially but was worried it was too verbose.

Reply:
Verbosity is my fav.

Reply:
Ok, done.